### PR TITLE
bump rancher-backup-restore-operator to 1.0.4-rc3

### PR DESCRIPTION
**Pull request by jiaqiluo**

### CODE REVIEW

**Review by nickgerace — APPROVED** [review]

**Comment by nickgerace** [comment]
Let's merge!

**Review by cbron — APPROVED** [review]
lgtm as long as the values.yaml image is correct, but ensure that tag exists before you merge

**Pull request merged**
